src/jgikbase/idmapping/core/namespace.py

```diff
@@ -1,17 +1,17 @@
-'''
+"""
```

Comment:
What extra meaning or utility do you get in having a `core` utility? Wouldn't "core" modules simply go in `idmapping/`
I'd also still argue in favor of having `src/idmapping/etc` instead of `src/jgikbase/idmapping` to simplify your directory structure. You won't have any namespace conflicts if you install dependencies using a package manager instead of injecting third-party code in your repo.